submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#" tabindex="1">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#" tabindex="2">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="3">
+                    <label for="popup-input-id" class="popup-btn">
+                        <img class="popup-icon" src="./icons/popup-button.png" alt="popup icon">
+                    </label>
+                    <div class="popup-container" tabindex="-1">
+                        <input id="more-input" class="more-input" type="checkbox" tabindex="16">
+                        <label for="more-input" class="more-btn">
+                        </label>
+                        <ul class="popup-list">
+                            <li class="popup-item">
+                                <a class="popup link" href="#" tabindex="4">
+                                    <img class="popup-img" src="./icons/calendar.png" alt="Calendar">
+                                    <p class="popup-name">Calendar</p>
+                                </a>
+                            </li>
+                            <li class="popup-item">
+                                <a class="popup link" href="#" tabindex="5">
+                                    <img class="popup-img" src="./icons/mail.png" alt="Mail">
+                                    <p class="popup-name">Mail</p>
+                                </a>
+                            </li>
+                            <li class="popup-item">
+                                <a class="popup link" href="#" tabindex="6">
+                                    <img class="popup-img" src="./icons/maps.png" alt="Map">
+                                    <p class="popup-name">Map</p>
+                                </a>
+                            </li>
+                            <li class="popup-item">
+                                <a class="popup link" href="#" tabindex="7">
+                                    <img class="popup-img" src="./icons/files.png" alt="Files">
+                                    <p class="popup-name">Files</p>
+                                </a>
+                            </li>
+                            <li class="popup-item">
+                                <a class="popup link" href="#" tabindex="8">
+                                    <img class="popup-img" src="./icons/tanslation.png" alt="Translate">
```

Comment:
In one of the previous commits, you remove value from `alt` attribute but now all icons have it.

Reply:
Removed all icon alt content